lib/ui/painting/codec.cc

```diff
@@ -75,7 +75,8 @@ fxl::RefPtr<Codec> InitCodec(sk_sp<SkData> buffer, size_t trace_id) {
 
   std::unique_ptr<SkCodec> skCodec = SkCodec::MakeFromData(buffer);
   if (!skCodec) {
-    FXL_LOG(ERROR) << "SkCodec::MakeFromData failed";
+    FXL_LOG(ERROR) << "Image data is either invalid, or not a supported "
+                      "format, decoding failed.";
```

Comment:
nits:
- "or it is encoded using an unsupported format"
- "Decoding image failed" would read better at the beginning as the user is more likely to be interested in what happened before the explanation of why it happened.

Reply:
PTAL